servers/rendering/renderer_viewport.cpp

```diff
@@ -1394,19 +1392,23 @@ void RendererViewport::viewport_set_force_motion_vectors(RID p_viewport, bool p_
 	Viewport *viewport = viewport_owner.get_or_null(p_viewport);
 	ERR_FAIL_NULL(viewport);
 
-	if (viewport->force_motion_vectors == p_force_motion_vectors) {
+	_viewport_set_force_motion_vectors(viewport, p_force_motion_vectors);
```

Comment:
Replace with private function that takes a `Viewport *` rather than RID, to avoid performing the lookup and check.